pagopa/io-app-design-system
# [IOPLT-171] Introduces `CodeInput` component

## Short description
This PR introduces `CodeInput` component

## How to test
Check NumberPad screen to view new `CodeInput` component features

https://github.com/pagopa/io-app-design-system/assets/3959405/ae9893d9-1629-4f1f-b0cf-5806dd74a9e3
